### PR TITLE
Updates for issues #81 #82 #83

### PR DESCRIPTION
**Pull request by anngvu**
Add more granular viability assay terms
Add new library kit value
Add new value for readStrandOrientation

### CODE REVIEW

File: NF.csv [hunk 1]
```diff
@@ -175,14 +179,14 @@ singleMolecule,An experiment in which a single molecule was used in an assay,,,F
...
...tochemistry, immunofluorescence, autoradiography, electrochemiluminescence, Real Time PCR, in vivo tumor growth, scale, Blood Chemistry Measurement, in vivo bioluminescence, immunohistochemistry, NOMe-Seq, FIA-MSMS, UPLC-MSMS, HPLC-MSMS, liquid chromatography-electrochemical detection, rnaSeq, mirnaSeq, LC-MSMS, LC-MS, lncrnaSeq, Whole Exome Sequencing, Sanger sequencing, ChIPSeq, rnaArray, snpArray, methylationArray, mirnaArray, bisulfiteSeq, ATACSeq, HI-C, errBisulfiteSeq, ISOSeq, westernBlot, wholeGenomeSeq, polymeraseChainReaction, cellViabilityAssay, atomicForceMicroscopy, brightfieldMicroscopy, tractionForceMicroscopy, nextGenerationTargetedSequencing, jumpingLibrary, MIB/MS, scCGIseq, MudPIT, questionnaire, DNA optical mapping, oxBS-Seq, Ribo-Seq, Positron Emission Tomography, RPPA, ELISA, LTP, LFP, whole-cell patch clamp, sandwich ELISA, Laser Speckle Imaging, Genotyping, TMT quantitation, label free mass spectrometry, cell count, reporter gene assay, Single Cell RNA Sequencing, Conventional MRI, Functional MRI, Magnetic Resonance Spectroscopy, 2D-Incucyte assay, 2D-cell titer glo assay, 3D microtissue viability assay, in vivo PDX viability assay",,TRUE,,experimentalData,,,
```
Comment (allaway) [inline NF.csv:182]:
```suggestion
assay,The technology used to generate the data in this file,"elevated plus maze test, open field test, rotarod performance test, mRNAcounts, high content screen, Immunocytochemistry, immunofluorescence, autoradiography, electrochemiluminescence, Real Time PCR, in vivo tumor growth, scale, Blood Chemistry Measurement, in vivo bioluminescence, immunohistochemistry, NOMe-Seq, FIA-MSMS, UPLC-MSMS, HPLC-MSMS, liquid chromatography-electrochemical detection, rnaSeq, mirnaSeq, LC-MSMS, LC-MS, lncrnaSeq, Whole Exome Sequencing, Sanger sequencing, ChIPSeq, rnaArray, snpArray, methylationArray, mirnaArray, bisulfiteSeq, ATACSeq, HI-C, errBisulfiteSeq, ISOSeq, westernBlot, wholeGenomeSeq, polymeraseChainReaction, cellViabilityAssay, atomicForceMicroscopy, brightfieldMicroscopy, tractionForceMicroscopy, nextGenerationTargetedSequencing, jumpingLibrary, MIB/MS, scCGIseq, MudPIT, questionnaire, DNA optical mapping, oxBS-Seq, Ribo-Seq, Positron Emission Tomography, RPPA, ELISA, LTP, LFP, whole-cell patch clamp, sandwich ELISA, Laser Speckle Imaging, Genotyping, TMT quantitation, label free mass spectrometry, cell count, reporter gene assay, Single Cell RNA Sequencing, Conventional MRI, Functional MRI, Magnetic Resonance Spectroscopy, 2D Incucyte assay, 2D CellTiter-Glo assay, 3D microtissue viability assay, in vivo PDX viability assay",,TRUE,,experimentalData,,,
```

File: NF.csv [hunk 2]
```diff
@@ -44,6 +44,10 @@ westernBlot,A multistep process in which a mixture of proteins is separated by g
 wholeGenomeSeq,Laboratory technique to sequence the complete DNA sequence of an organism's genome at a single time,,,FALSE,http://edamontology.org/topic_3673,assay,,,
 polymeraseChainReaction,"A rapid technique for in vitro amplification of specific DNA or RNA sequences, allowing small quantities of short sequences to be analyzed without cloning",,,FALSE,http://purl.obolibrary.org/obo/MMO_0000459,assay,,,
 cellViabilityAssay,This assay type measures the cellular state of living or dying by measuring an indicator of life or death,,,FALSE,http://www.bioassayontology.org/bao#BAO_0003009,assay,,,
+2D-Incucyte assay,,,,FALSE,,assay,,,
```
Comment (allaway) [inline NF.csv:47]:
```suggestion
2D Incucyte assay,,,,FALSE,,assay,,,
```

File: NF.csv [hunk 3]
```diff
@@ -44,6 +44,10 @@ westernBlot,A multistep process in which a mixture of proteins is separated by g
...
+2D-cell titer glo assay,,,,FALSE,https://www.promega.com/products/cell-health-assays/cell-viability-and-cytotoxicity-assays/celltiter_glo-2_0-assay/?catNum=G9241,assay,,,
```
Comment (allaway) [inline NF.csv:48]:
```suggestion
2D CellTiter-Glo assay,,,,FALSE,https://www.promega.com/products/cell-health-assays/cell-viability-and-cytotoxicity-assays/celltiter_glo-2_0-assay/?catNum=G9241,assay,,,
```

**Review by allaway — DISMISSED** [review]
I suggested some changes for consistency and formatting of brand names.

One other general comment is whether we should remove "assay" from the "assay" values eg `2D CellTiter-Glo assay` since it's a little redundant, but I don't know if that matters all that much.

**Comment by allaway** [comment]
I just realized that my changes in the csv wouldn't be reflected in the json-ld, so I dismissed my approval :) 

Let me know if these changes look good to you and then if you can regenerate the json-ld, I'll approve the changes. thanks!

**Review by allaway — APPROVED** [review]
LGTM, thanks!

**Comment by anngvu** [comment]
Thanks for the review! Definitely helps these terms to be cleaner. Applied all review formatting suggestions and added 2 commits, the first in consideration of
> general comment is whether we should remove "assay" from the "assay" values

and the second to take care of 

> regenerate the json-ld

**Pull request merged**